scripts/package.json

```diff
@@ -47,7 +47,7 @@
     "@types/puppeteer": "1.12.3",
     "@types/read-pkg-up": "^6.0.0",
     "async": "^2.6.1",
-    "autoprefixer": "^10.2.1",
+    "autoprefixer": "^9.7.2",
```

Comment:
you can remove this , as it's now defined via single version policy within root `package.json`